__tests__/App.test.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from '../src/App';
+
+ it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<App />, div);
+    ReactDOM.unmountComponentAtNode(div);
+});
```

Comment:
Hello @Bakley ,
Please add a new  line at the end of the file.

Reply:
Done that